apps/omg_watcher_info/test/omg_watcher_info/db/block_test.exs

```diff
@@ -17,80 +17,125 @@ defmodule OMG.WatcherInfo.DB.BlockTest do
   use ExUnit.Case, async: false
   use OMG.Fixtures
 
+  import OMG.WatcherInfo.Factory
+
+  alias OMG.Utils.Paginator
   alias OMG.WatcherInfo.DB
 
   @eth OMG.Eth.RootChain.eth_pseudo_address()
 
-  describe ":initial_blocks fixture" do
-    @tag fixtures: [:initial_blocks]
-    test "preserves blocks in DB" do
-      assert [
-               %DB.Block{blknum: 1000, eth_height: 1, hash: "#1000"},
-               %DB.Block{blknum: 2000, eth_height: 1, hash: "#2000"},
-               %DB.Block{blknum: 3000, eth_height: 1, hash: "#3000"}
-             ] = DB.Repo.all(DB.Block)
-    end
-  end
-
   describe "get_max_blknum/0" do
     @tag fixtures: [:phoenix_ecto_sandbox]
     test "last consumed block is not set in empty database" do
       assert nil == DB.Block.get_max_blknum()
     end
 
-    @tag fixtures: [:initial_blocks]
+    @tag fixtures: [:phoenix_ecto_sandbox]
     test "last consumed block returns correct block number" do
+      _ = insert(:block, blknum: 1000, hash: <<1000>>, eth_height: 1, timestamp: 100)
+      _ = insert(:block, blknum: 2000, hash: <<2000>>, eth_height: 2, timestamp: 200)
+      _ = insert(:block, blknum: 3000, hash: <<3000>>, eth_height: 3, timestamp: 300)
+
       assert 3000 == DB.Block.get_max_blknum()
     end
   end
 
+  describe "get_blocks/1" do
+    @tag fixtures: [:phoenix_ecto_sandbox]
+    test "returns a list of blocks" do
+      _ = insert(:block, blknum: 1000, hash: "0x1000", eth_height: 1, timestamp: 100)
+      _ = insert(:block, blknum: 2000, hash: "0x2000", eth_height: 2, timestamp: 200)
+      _ = insert(:block, blknum: 3000, hash: "0x3000", eth_height: 3, timestamp: 300)
+
+      paginator = %Paginator{
+        data: [],
+        data_paging: %{
+          limit: 10,
+          page: 1
+        }
+      }
+
+      results = DB.Block.get_blocks(paginator)
+
+      assert length(results.data) == 3
```

Comment:
Perhaps assert that the array items are `%DB.Block{}`

Reply:
Added in 298831f8761d62863e2c627977cd894b928bedef